samples/net/mdns_responder/src/service.c

```diff
@@ -19,11 +19,19 @@ LOG_MODULE_REGISTER(mdns_echo_service, LOG_LEVEL_DBG);
 /* A default port of 0 causes bind(2) to request an ephemeral port */
```

Comment:
Commit subject of `samples: net: mdns_responder: Fix coverity issue` is ok, but please change the commit message, so something like this:
```
Check return value if sendto() fails.

Coverity-CID: 215391
Fixes #33093

Signed-off-by: ....
```

Reply:
Fixed